### PR TITLE
Creating KafkaConsumer w/ types

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

**Comment by matzew** [comment]
@dimitraz can you check if this helps w/ the generic SerDe ?

**Comment by matzew** [comment]
I think I will make it a bit nicer ... but just wondering if that actually helps ya

**Comment by dimitraz** [comment]
@matzew I'm very reluctant to say it's working, but I think it actually might be.. Will carry out a few more tests to be certain

File: src/main/java/net/wessendorf/kafka/impl/DelegationKafkaConsumer.java [hunk 1]
```diff
@@ -76,21 +77,28 @@ public DelegationKafkaConsumer() {
         }
     }
 
+    private <K, V> void createKafkaConsumer(final Class<K> type, final Class<V> val, final Properties consumerProperties) {
```
Comment (dimitraz) [inline src/main/java/net/wessendorf/kafka/impl/DelegationKafkaConsumer.java]: `final Class<K> keyType` and `final Class<V> valType` is probably nicer?

Comment (dimitraz) [inline src/main/java/net/wessendorf/kafka/impl/DelegationKafkaConsumer.java]: or `keyTypeClass` etc

**Comment by matzew** [comment]
@dimitraz yeah, changed that.

**Pull request merged**